xapian-letor/api/featurevector_internal.cc

```diff
@@ -117,3 +117,31 @@ FeatureVector::Internal::get_nonzero_num(){
     //cout << "nonzero: "<< nonzero <<endl;
     return nonzero;
 }
+
+void
+FeatureVector::Internal::normalise() {
+
+    int num_features = 19;
+    double temp = 0.0;
+    double max[num_features];
+
+    for(int i=0; i<19; ++i)
```

Comment:
Indentation seems missing here, but it's probably clearest to wrap as an actual block.